src/components/dashboard.js

```diff
@@ -121,105 +111,100 @@ export default function Dashboard(props) {
+        open={open}
+        handleDrawerClose={handleDrawerClose}
+      />
     </div>
   );
 }
 
 function UpperBar({ renderDrawer, classes, open, handleDrawerOpen }) {
-  return (
-    <AppBar position="absolute" className={clsx(classes.appBar, open && classes.appBarShift)}>
-      <Toolbar className={classes.toolbar}>
-        {renderDrawer &&
+  const [anchorEl, setAnchorEl] = React.useState(null);
+  const handleMenu = (event) => {
+    setAnchorEl(event.currentTarget);
+  };
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+  const profileOpen = Boolean(anchorEl);
+  const renderElement = () => {
+    if (renderDrawer) {
+      return (
+        <div>
+          {/* TODO: Change <span> to <Typography> */}
+          <span>
+            DISPLAY NAME
+          </span>
           <IconButton
-            edge="start"
+            aria-label="account of current user"
+            aria-controls="menu-appbar"
+            aria-haspopup="true"
+            onClick={handleMenu}
             color="inherit"
-            aria-label="open drawer"
-            onClick={handleDrawerOpen}
-            className={clsx(classes.menuButton, open && classes.menuButtonHidden)}
           >
-            <MenuIcon />
-          </IconButton>
-        }
-        <Typography component="h1" variant="h6" color="inherit" noWrap className={classes.title}>
-          DASHBOARD
-        </Typography>
-        <RouterLink to="/signup">
-          <Typography component="h1" variant="h6" color="inherit" noWrap className={classes.title}>
-            SIGN UP
-          </Typography>
-        </RouterLink>
-        <RouterLink to="/login">
-          <Typography component="h1" variant="h6" color="inherit" noWrap className={classes.title}>
-            LOG IN
-          </Typography>
-        </RouterLink>
-      </Toolbar>
-    </AppBar>
-  );
-}
-
-function LeftDrawer({ renderDrawer, classes, open, handleDrawerClose }) {
-  if (renderDrawer) {
-    return (
-      <Drawer
-        variant="permanent"
-        classes={{
-          paper: clsx(classes.drawerPaper, !open && classes.drawerPaperClose),
-        }}
-        open={open}
-      >
-       <div className={classes.toolbarIcon}>
-          <IconButton onClick={handleDrawerClose}>
-            <ChevronLeftIcon />
+            <AccountCircle />
           </IconButton>
+          <Menu
+            id="menu-appbar"
+            anchorEl={anchorEl}
+            anchorOrigin={{
+              vertical: 'top',
+              horizontal: 'right',
+            }}
+            keepMounted
+            transformOrigin={{
+              vertical: 'top',
+              horizontal: 'right',
+            }}
+            open={profileOpen}
+            onClose={handleClose}
+          >
+            <MenuItem onClick={handleClose}>Profile</MenuItem>
+            <MenuItem onClick={handleClose}>My account</MenuItem>
+          </Menu>
+        </div>
+      );
+    }
+    else {
+      return (
```

Comment:
`} else {`

Reply:
We will just use Prettier for code consistency, maybe before we deploy our final product.